src/main/content/_assets/css/post.scss

```diff
@@ -507,4 +507,10 @@ min-width: 100px; }
 #article_container table code, #article_container table pre {
     background-color: inherit;
     color: inherit;
+}
+
+iframe{
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
```

Comment:
Since we are editing this file, please go ahead and add a end of line to correct the minor file warning